sdk/tables/azure-data-tables/azure/data/tables/_entity.py

```diff
@@ -75,17 +76,40 @@ class EntityProperty(object):
     """
 
     def __init__(self,
+                value=None, # type: any
                  type=None,  # type: Union[str,EdmType] # pylint:disable=W0622
-                 value=None  # type: Any
-                 ):
+                #  value=None  # type: Any
```

Comment:
should remove this comment

Reply:
Also out of curiosity: what was the reasoning behind flipping the oreder

Reply:
Most of the time the type isn't inputted, it was cleaner in my mind to have the most common parameter first.